### PR TITLE
gh-112306: Amend 3.7.rst

### PR DESCRIPTION
**Pull request by nineteendo**
* Issue: gh-112306

----
📚 Documentation preview 📚: https://cpython-previews--119160.org.readthedocs.build/

### CODE REVIEW

**Comment by nineteendo** [comment]
I'm a little confused as to why it hasn't been implemented for 3.13 yet.

**Comment by hauntsaninja** [comment]
I think Larry didn't have time to push it through (and we're now at beta, so will need to wait for 3.14). Jelle and Łukasz have both said they'll ensure it gets done for 3.14.

File: Doc/whatsnew/3.7.rst [hunk 1]
```diff
@@ -171,7 +171,11 @@ on a per-module basis in Python 3.7 using a :mod:`__future__` import::
 
     from __future__ import annotations
 
-It will become the default in Python 3.10.
+It was previously scheduled to become mandatory in Python 3.10, but
+the Python Steering Council decided to delay the change until 3.13
```
Comment (nineteendo) [inline Doc/whatsnew/3.7.rst]:
```suggestion
the Python Steering Council decided to delay the change until 3.14
```

**Comment by nineteendo** [comment]
OK, I've updated it to 3.13. Doesn't that mean the PEP also needs to be updated? It could be quite confusing to read it has been implemented in 3.13 when it hasn't.

**Comment by nineteendo** [comment]
cc @JelleZijlstra

**Comment by JelleZijlstra** [comment]
This isn't entirely accurate, since `from __future__ import annotations` as originally written definitely will *not* become the default. However, I'm not entirely clear yet on what the end state will be; just posted about it at https://discuss.python.org/t/pep-649-deferred-evaluation-of-annotations-tentatively-accepted/21331/44.

**Comment by nineteendo** [comment]
I'll update this when a decision has been made.

**Comment by nineteendo** [comment]
@JelleZijlstra, does this look better? Or should be the Steering Council be mentioned in the note?

**Comment by nineteendo** [comment]
Does this look better? `from __future__ import annotations` will either continue to work, become a no-op or be removed. It will never become the default.

**Comment by nineteendo** [comment]
I'm closing this. Could you assign yourself to the issue?

**Pull request closed without merging**